spinalcordtoolbox/reports/qc.py

```diff
@@ -419,15 +522,25 @@ def __init__(self, input_file, command, args, orientation, dest_folder, dpi=300,
         self.root_folder = dest_folder
         self.mod_date = datetime.datetime.strftime(datetime.datetime.now(), '%Y_%m_%d_%H%M%S.%f')
         self.qc_results = os.path.join(dest_folder, '_json/qc_' + self.mod_date + '.json')
-        self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.png')
-        self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.png')
+        if command in ['sct_fmri_moco', 'sct_dmri_moco']:
+            self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.gif')
+            self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.gif')
+        else:
+            self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.png')
+            self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.png')
 
     def abs_bkg_img_path(self):
         return os.path.join(self.root_folder, self.bkg_img_path)
 
     def abs_overlay_img_path(self):
         return os.path.join(self.root_folder, self.overlay_img_path)
 
+    def abs_bkg_gif_path(self):
+        return os.path.join(self.root_folder, self.bkg_img_path)
+
+    def abs_overlay_gif_path(self):
+        return os.path.join(self.root_folder, self.overlay_img_path)
+
```

Comment:
In lines 538-543, do we actually need new attributes `abs_bkg_gif_path` and `abs_overlay_gif_path`? Aren't they identical to `abs_bkg_img_path` and `abs_overlay_img_path`?

The distinction between `.gif` and `.png` is already made earlier on when `self.bkg_img_path` and `self.overlay_img_path` are first set, right? (lines 525-530)

Reply:
You're right, they're not necessary anymore, thanks for pointing this out! I was using them in a first version where I was saving all the indivual frames as png images, but I didn't think to remove `abs_bkg_gif_path` and `abs_overlay_gif_path` after making the change!